Renegade X Launcher/VersionCheck.cs

```diff
@@ -138,24 +139,27 @@ public static void UpdateGameVersion()
                 LauncherPatchHash = versionData["launcher"]["patch_hash"];
 
                 // Game
-                LatestGameVersion = new Version
+                _latestGameVersion = new Version
                 {
                     Name = versionData["game"]["version_name"],
                     Number = versionData["game"]["version_number"],
                 };
                 InstructionsHash = versionData["game"]["instructions_hash"];
                 GamePatchPath = versionData["game"]["patch_path"];
-                //GamePatchUrls = versionData["game"]["patch_urls"].ToObject<string[]>();
-                GamePatchUrls = UpdateServerModelFactory.CreateUpdateServerModels(versionData["game"]["patch_urls"].ToObject<string[]>());
+
+                // Server URL's list & Friendly Names
+                foreach (var x in versionData["game"]["server_urls"].ToObject<dynamic>())

```

Comment:
Since it might not be exclusively urls in the future, maybe we should change this to just `mirrors` at some point